meetings/2025-02-26.md

```diff
@@ -28,16 +32,73 @@
+* Michal: I have an impression that SPARQL Update is not a part of the specification. If it is not I would find it confusing to be recommended. I did the move from SPARQL to N3 in some of my work.
+* ML: I'm not against SPARQL-update. I think N3 patch has been introduced to resolve issues that were found in SPARQL-update (different interpretations). We may need to detail SPARQL-update more to also address those issues (so everyone use the SPARQL-update in a consistent way). 
+* JW: With Ruben I will only clarify some details and possible misunderstanding. It would be good if someone else would go throug the specification. If it is there we can create PR in with suggested note. If it is not there, we need to procede with the current PR. There is a question of raising a note to say that SPARQL Update may be reintroduced. In both cases we should cc PAC. 
+* eP: I believe SPARQL update is not in the spec. Since Ruben was working on N3 patch, I assume he's fully aware of the issue Maxime is raising. I assume Ruben should have a good recollection of the reason for dropping SPARQL update.
+* JW: The reason to drop SPARQL update was related to HTTP responses, even if you have append only access. Which would leak information. Custom response codes could be use with N3 Patch. We could also use SPARQL Update with 
+* eP: woudl it be possible to use SPARQL update with a different/customized media type.
+* JW: sounds like a good idea, bring it up.
+* ML: I looked into the source code of CSS, it is only 3 lines that need to be changed to allow blank nodes. See https://github.com/CommunitySolidServer/CommunitySolidServer/blob/e5e7b95327f18b032742d8414e0529913a6ba4de/src/http/input/body/N3PatchBodyParser.ts#L124
+* HZ: I think we have a path forward.
+
 ### Group Based Access Policies
 
+* eP: Simple diagram with Organizations, Roles/Teams, and People
+
+![image](https://hackmd.io/_uploads/SkcouAockl.png)
+
+* HZ: It was discussed during last Solid Practicioners meeting. My opinion changes every time, there are issues related to an information leak. I think we agreed to have a separate STM. I remember that Sebastian from ActivityPods
+* eP: if the authorization system does not have access to the organization specific teams/groups, it cannot efficiently enforce authorization. I think we should differentiate membership from autonomous organization from roles within an org.
+* eP: Yesterday also Christoph preesnted their work https://publikationen.bibliothek.kit.edu/1000174700/154929747
+* HZ: I think proxies can be a good think, it breaks the audit trail. 
+* Michal: I have another issue with the gruop. When the vcard group is colocated with the resources, then it doesn't need to be public. This would mean that the list would be distributed and copied. I also have other issues like scalability.
```

Comment:
```suggestion
* eP: Also yesterday, Christoph presented their work https://publikationen.bibliothek.kit.edu/1000174700/154929747
* HZ: I think proxies can be a good thin. They break the audit trail. 
* Michal: I have another issue with the group. When the vcard group is colocated with the resources, it doesn't need to be public. This would mean that the list would be distributed and copied. I also have other issues like scalability.
```